docs/timezone.md

```diff
@@ -1,19 +1,213 @@
+
+Example usage:
+
+```javascript
+// Converting a specific absolute time to a calendar date / wall-clock time
+timestamp = new Temporal.Absolute(1553993100000000000n);
+tz = new Temporal.TimeZone('Europe/Berlin');
+tz.getDateTimeFor(timestamp);  // => 2019-03-31T01:45
+
+// What time was the Unix Epoch (timestamp 0) in Bell Labs (Murray Hill, New Jersey, USA)?
+epoch = new Temporal.Absolute(0n);
+tz = new Temporal.TimeZone('America/New_York');
+tz.getDateTimeFor(epoch);  // => 1969-12-31T19:00
+```
+
 ## timeZone.getAbsoluteFor(dateTime: Temporal.DateTime, disambiguation: 'earlier' | 'later' | 'reject' = 'earlier') : Temporal.Absolute
 
+**Parameters:**
+- `dateTime` (`Temporal.DateTime`): A calendar date and wall-clock time to convert.
+- `disambiguation` (optional `string`): How to disambiguate if the wall-clock time given by `dateTime` can refer to more than one absolute time.
+  Allowed values are `earlier`, `later`, and `reject`.
+  The default is `earlier`.
+
+**Returns:** A `Temporal.Absolute` object indicating the absolute time in `timeZone` at the time of the calendar date and wall-clock time from `dateTime`.
+
+This method is one way to convert a `Temporal.DateTime` to a `Temporal.Absolute`.
+
+### Resolving ambiguity
+
+> This explanation was adapted from the [moment-timezone documentation](https://github.com/moment/momentjs.com/blob/master/docs/moment-timezone/01-using-timezones/02-parsing-ambiguous-inputs.md).
+
+This conversion is not necessarily one-to-one.
+Due to DST time changes, there is a possibility that a wall-clock time either does not exist, or has existed twice.
+The `disambiguation` argument controls what absolute time to return in the case of ambiguity:
+- `earlier` (the default): The earlier of the two possible absolute times will be returned.
+- `later`: The later of the two possible absolute times will be returned.
+- `reject`: A `RangeError` will be thrown.
+
+When entering DST, clocks move forward an hour.
+In reality, it is not time that is moving, it is the offset moving.
+Moving the offset forward gives the illusion that an hour has disappeared.
+As the clock ticks, you can see it move from 1:58 to 1:59 to 3:00.
+It is easier to see what is actually happening when you include the offset.
+
+```
+1:58 +1
+1:59 +1
+3:00 +2
+3:01 +2
+```
+
+The result is that any time between 1:59:59 and 3:00:00 never actually happened.
+In `earlier` mode, the absolute time will skip backwards by the amount of the DST gap (usually 1 hour).
+In `later` mode, the absolute time will skip forwards by the amount of the DST gap.
+
+```javascript
+tz = new Temporal.TimeZone('Europe/Berlin');
+dt = new Temporal.DateTime(2019, 3, 31, 2, 45);
+tz.getAbsoluteFor(dt, 'earlier');  // => 2019-03-31T00:45Z
+tz.getAbsoluteFor(dt, 'later');    // => 2019-03-31T01:45Z
+tz.getAbsoluteFor(dt, 'reject');   // throws
+```
+
+In this example, the wall-clock time 2:45 doesn't exist, so it is treated as either 1:45 +01:00 or 3:45 +02:00.
```

Comment:
This seems wrong to me... it means that `tz.getAbsoluteFor(skipped_02_45.minus("PT50M"), "earlier")` is _later_ than `tz.getAbsoluteFor(skipped_02_45, "earlier")`! Shouldn't the "two possible absolute times" for a time of day skipped by a positive transition be "immediately before the transition" vs. "the matching time in the pre-transition offset" (e.g., converting a skipped **02:45** to **01:59:59.999999999** vs. **03:45**)?

Reply:
I don't have an opinion yet on which way is correct, but FWIW the behaviour described here is what the polyfill currently does. (The spec is marked TODO on this point.) The moment-timezone package also works like this. So maybe we should open an issue to resolve that question separately.

Reply:
+1 to following up on this.

Reply:
Opened https://github.com/tc39/proposal-temporal/issues/317 to follow up.